flows/triggers/user.go

```diff
@@ -0,0 +1,62 @@
+package triggers
+
+import (
+	"encoding/json"
+	"time"
+
+	"github.com/nyaruka/goflow/flows"
+	"github.com/nyaruka/goflow/utils"
+)
+
+// TypeUser is a constant for incoming messages
+const TypeUser string = "user"
+
+// UserTrigger is used when a session was triggered manually by a user
```

Comment:
I think having this mention "user" probably doesn't make sense. Seems this is more just a "manual" or "explicit" trigger.. outside systems may not have any concept of a user, nor does this describe the user in any way, so the intrinsic piece to this object is not the user but that it is manual.